auth0/src/main/java/com/auth0/android/util/Telemetry.java

```diff
@@ -50,7 +49,7 @@ public Telemetry(String name, String version, String libraryVersion) {
         }
         values.put(ENV_KEY, env);
         String json = new Gson().toJson(values);
-        Charset utf8 = StandardCharsets.UTF_8;
+        Charset utf8 = Charset.forName("UTF-8");
```

Comment:
Any reason not to use `Charset.defaultCharset()`? Which will default to UTF-8 on Android https://developer.android.com/reference/java/nio/charset/Charset

Reply:
I could, but the goal was to only change imports. This change was a mistake from lint auto-correcting stuff.